packages/contentstack-export-to-csv/src/util/index.js

```diff
@@ -823,12 +840,61 @@ async function getTeamsDetail(allTeamsData, organization ,teamUid) {
     const fileName = `${kebabize(
       organization.name.replace(config.organizationNameRegex, ''),
     )}_team_${teamUid}_User_Details_export.csv`;
-    
+
     write(this, team.users, fileName, 'Team User details');
-  
   }
 }
 
+async function exportRoleMappings(managementAPIClient, allTeamsData, teamUid) {
+  let stackRoleWithTeamData = [];
+  if (teamUid) {
+    const team = allTeamsData.filter((team) => team?.uid === teamUid)[0];
+    for (const stack of team?.stackRoleMapping) {
+      const roleData = await mapRoleWithTeams(managementAPIClient, stack, team?.name, team?.uid);
+      stackRoleWithTeamData.push(...roleData);
+    }
+  } else {
+    for (const team of allTeamsData) {
+      for (const stack of team?.stackRoleMapping) {
```

Comment:
We may possibly get `undefined is not iterable (cannot read property Symbol(Symbol.iterator))` error

Try to handle it like `for (const stack of team?.stackRoleMapping ?? [])`

NOTE: The same case may also apply to the parent loop. Just review it once.

Reply:
Checked both and modified the code